gradle-consistent-versions-idea-plugin/src/main/java/com/palantir/gradle/versions/intellij/VersionCompletionContributor.java

```diff
@@ -104,52 +113,82 @@ private void addLoadingElement(CompletionResultSet sortedResultSet) {
     }
 
     private void handleDependencyWithoutWildcard(
```

Comment:
We need to handle these as a chunk so that we can be sure that there are no versions for all repos rather than just no versions for one repo